.github/workflows/ci.yml

```
@@ -12,15 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        redis-version:
+          - 4.0
+          - 4.1
+          - 4.2
+          - 4.3
+          - 4.4
+          - 4.5
+          - 4.6
+          - latest
         ruby-version:
           - 2.3
           - 2.4
           - 2.5
           - 2.6
           - 2.7
-          - "3.0"
+          - 3.0
```

Has https://github.com/actions/runner/issues/849 been fixed?

At least in the past, it was better to keep the quotes.  Otherwise, it's treated as a number instead of a version string.  And `3.0` can be interpreted as `3`, which can mean "the latest 3.x version".  If the redis gem ever has a "4.10" version that could be interpreted as "4.1", etc.  If consistency is what's wanted, you can add quotes to *all* of the versions.

```suggestion
          - "3.0"
```

---

You are right, figured this was fixed, but it isn't: https://github.com/resque/resque/runs/7859272121?check_suite_focus=true